src/frontend/screens/Settings/sections/GamesSettings/index.scss

```diff
@@ -86,3 +86,7 @@ details {
     gap: 10px;
   }
 }
+
+.Field label {
+  float: left;
```

Comment:
maybe use
```css
.Field {
  text-align: left
}
```

`float: left` looks hacky

Reply:
Thanks for your feedback @arielj I'm not very familiar with CSS. Just pushed a new commit with your suggestion.